File: data/python/SMALLEST_POWER_OF_2_GREATER_THAN_OR_EQUAL_TO_N_MAIN.py
if __name__ == '__main__':
    param = [
    (13,),
    (27,),
    (1,),
    (24,),
    (98,),
    (94,),
    (36,),
    (41,),
    (74,),
    (39,)
        ]
    n_success = 0
    print("class,mutant,test_index,result\n")
    for i, parameters_set in enumerate(param):
        try:
            print("SMALLEST_POWER_OF_2_GREATER_THAN_OR_EQUAL_TO_N," + sys.argv[1] + "," + str(i) + "," + str(f_gold(*parameters_set)))
        except:
            print("SMALLEST_POWER_OF_2_GREATER_THAN_OR_EQUAL_TO_N," + sys.argv[1] + "," + str(i) + ",EXCEPTION")

File: data/java/SMALLEST_POWER_OF_2_GREATER_THAN_OR_EQUAL_TO_N_MAIN.java
public static void main(String args[]) throws IOException {
    int n_success = 0;
    List<Integer> param0 = new ArrayList<>();
    param0.add(13);
    param0.add(27);
    param0.add(1);
    param0.add(24);
    param0.add(98);
    param0.add(94);
    param0.add(36);
    param0.add(41);
    param0.add(74);
    param0.add(39);
    StringBuilder builder = new StringBuilder();
    builder.append("class,mutant,test_index,result\n");
    FileWriter writer = new FileWriter(args[0]);
    for(int i = 0; i < param0.size(); ++i)
    {
        try {
            builder.append("SMALLEST_POWER_OF_2_GREATER_THAN_OR_EQUAL_TO_N," + args[1] + "," + i + "," + f_gold(param0.get(i)) + "\n");
        } catch (Exception e) {
            builder.append("SMALLEST_POWER_OF_2_GREATER_THAN_OR_EQUAL_TO_N," + args[1] + "," + i + ",EXCEPTION\n");
        }
    }
    writer.write(builder.toString());
    writer.close();
}
